src/app/work-item/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -36,20 +38,58 @@ export class WorkItemQuickAddComponent implements OnInit {
     } as WorkItem;
   }
 
-  save(): void {
+  //save(): void {
```

Comment:
I think you need the code that you commented out here. That at least allows it to save and the enter button saves too.